index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="index.css">
+  <title>Awesome books</title>
+</head>
+
+<body>
+  <header>
+  <nav>
+    <div><a class="badge" href="index.html">Awesome Books</a></div>
+
+    <ul class="menu">
+
+      <li>
+        <a href="#list" id="list-link">List</a>
+      </li><span>|</span>
+      <li>
+        <a href="#add" id="add-link">Add</a></li>
+        <span>|</span>
+      <li>
+        <a href="#contact" id="contact-link">Contact</a>
+      </li>
+       
+  </nav>
+  </header>
+  
+  <div class="date" id="date"></div>
+
+  <section class="list-section" id="list">
```

Comment:
- [ ] Kindly make sure this page (section) is visible by default. Currently, I have to click on the `List` to render this section. It'll make more sense if this is visible by default.